### PR TITLE
Retrieve fields documentation from DataCore

### PR DESCRIPTION
**Pull request by geraldgole**

### CODE REVIEW

File: src/main/resources/public/jsx/DatasetForm.jsx [hunk 1]
```diff
@@ -22,8 +22,7 @@ class DatasetForm extends React.Component {
         organizations:React.PropTypes.array.isRequired,
         datasetName: React.PropTypes.string.isRequired,
         onChangeNotif: React.PropTypes.func.isRequired,
-        geoLocation: React.PropTypes.object.isRequired,
-        private : React.PropTypes.bool.isRequired
```
Comment (bobeal) [inline src/main/resources/public/jsx/DatasetForm.jsx]: does not seem normal ?

File: build.gradle [hunk 2]
```diff
@@ -51,10 +50,10 @@ dependencies {
 	compile('org.springframework.boot:spring-boot-starter-websocket')
 	compile('joda-time:joda-time:2.9.9')
 	compile('io.javaslang:javaslang:2.0.4')
-    compile('com.ozwillo:ozwillo-java-spring-integration:1.25.7')
+    compile('com.ozwillo:ozwillo-java-spring-integration:1.26.0-dev')
```
Comment (bobeal) [inline build.gradle]: there is nothing to add to use jitpack ?

File: src/main/resources/public/jsx/DatasetForm.jsx (hunk 1 above)
Comment (geraldgole) [reply src/main/resources/public/jsx/DatasetForm.jsx]: I just removed private property that was not mandatory.
geoLocation is kept.

Comment (geraldgole) [reply src/main/resources/public/jsx/DatasetForm.jsx]: Rollback : private property is set to required
`private` is provided with default value at DatasetForm instantiation (see last commit)

**Pull request merged**